Sources/RInAppMessaging/Extensions/UIFont+IAM.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit.UIFont
+
+/// Fonts referenced here must be pre-registered by the host application
+internal extension UIFont {
+    class func iamRegular(ofSize fontSize: CGFloat) -> UIFont {
```

Comment:
Do you use `class func` in order to override `iamRegular` function?
If not, can you use `static func`?